src/views/index_view.js

```diff
@@ -28,6 +33,10 @@ export default BaseView.extend({
     this.renderSubview(new IconView());
```

Comment:
...how about just `setTimeout(this.adjustHeight, 0)` after rendering the subviews?